pandas/io/formats/style.py

```diff
@@ -813,7 +900,7 @@ def hide_columns(self, subset):
         Parameters
         ----------
         subset: IndexSlice
-            An argument to ``DataFrame.loc`` that identifies which columns
```

Comment:
@jorisvandenbossche Do we have an official policy on single vs. double backticks? I know numpydoc recommends single, but I thought we used double so that it looked nicer in the HTML docs.